### DISCUSSION TITLE
How to specify the code to run on a particular gpu device

### DISCUSSION THREAD

**Discussion by ikun0124** [open-edge-platform/anomalib#1030] [Q&A]
I want to run a code on gpu 0 and another code on gpu 1, how should I set up the config file？

**Comment by ikun0124** [comment]
need help.......

**Comment by ikun0124** [comment]
need help.......

**Comment by samet-akcay** [comment]
Hi @loyal-ikun, you could refer to this link regarding configuring the GPUs. 
https://lightning.ai/docs/pytorch/stable/accelerators/gpu_basic.html#train-on-gpus

Based on this, you will need to modify [`accelerator`](https://github.com/openvinotoolkit/anomalib/blob/main/src/anomalib/models/padim/config.yaml#L93) and [`devices`](https://github.com/openvinotoolkit/anomalib/blob/main/src/anomalib/models/padim/config.yaml#L75) arguments in `Trainer` configurations in a [config](https://github.com/openvinotoolkit/anomalib/blob/main/src/anomalib/models/padim/config.yaml) file.

If you have any more questions that you cannot find in the documentation, let us know.
